src/meshdb/celery.py

```diff
@@ -1,11 +1,12 @@
 import os
 from pathlib import Path
-
+from django.conf import settings
 from celery import Celery, bootsteps
 from celery.apps.worker import Worker
 from celery.signals import beat_init, worker_ready, worker_shutdown
 from datadog import initialize, statsd
 from dotenv import load_dotenv
+import environment
```

Comment:
Consider only importing what you need.
```suggestion
from environment import CELERY_BROKER
```